### DISCUSSION TITLE
How to show data in Pivot Grid?

### DISCUSSION THREAD

**Discussion by georgemaka** [TabularEditor/TabularEditor3#635] [🙋♀️TE3 Q&A] [answered]
Why does pivot grid not show any data? The same columns/rows that I used in Power BI are the same that I am using in Tabular Editor. Power BI is showing the values, but nothing in Tabular Editor.

Also, tabular editor does not allow to add the data values of a column? Can I only add a measure for data?

**Comment by otykier** [comment, marked as answer]
Answered in #633
